core/templates/pages/classroom-page/classroom-page.component.ts

```diff
@@ -95,6 +100,16 @@ export class ClassroomPageComponent {
   getStaticImageUrl(imagePath: string): string {
     return this.urlInterpolationService.getStaticImageUrl(imagePath);
   }
+
+  // Used for choosing from classroom name or topic translation key to be
```

Comment:
This method is used to choose whether to display the classroom name or the topic translation key in the ...

Reply:
Done!